server/src/test/java/io/crate/integrationtests/LogicalReplicationITestCase.java

```diff
@@ -132,24 +133,50 @@ public void setupClusters() throws IOException, InterruptedException {
 
     @After
     public void clearCluster() throws Exception {
-        try {
-            publisherCluster.beforeIndexDeletion();
-            publisherCluster.assertSeqNos();
-            publisherCluster.assertSameDocIdsOnShards();
-            publisherCluster.assertConsistentHistoryBetweenTranslogAndLuceneIndex();
-
-            subscriberCluster.beforeIndexDeletion();
-            subscriberCluster.assertSeqNos();
-            subscriberCluster.assertSameDocIdsOnShards();
-            subscriberCluster.assertConsistentHistoryBetweenTranslogAndLuceneIndex();
-        } finally {
-            publisherCluster.wipe(Collections.emptySet());
-            publisherCluster.close();
-            subscriberCluster.wipe(Collections.emptySet());
-            subscriberCluster.close();
+        stopSubscriberCluster();
+        stopPublisherCluster();
+    }
+
+    protected void stopPublisherCluster() throws Exception {
```

Comment:
Minor: probably can be replace by a single stopCluster with cluster as a parameter.